plugins/modules/system/ssh_config.py

```diff
@@ -0,0 +1,259 @@
+    default: present
+    choices: [ 'present', 'absent' ]
+    type: str
+  user:
+    description:
+      - Which user account this configuration file belongs to.
+      - If none given, C(/etc/ssh/ssh_config) is used.
+      - If a user is given, C(~/.ssh/config) is used.
+    default: root
+    type: str
+  group:
+    description:
+      - Which group this configuration file belongs to.
+      - If non given, I(user) is used.
+    default: root
+    type: str
+  host:
+    description:
+      - The endpoint this configuration is valid for.
+      - Can be an actual address on the internet or an alias that will
+        connect to the value of I(hostname).
+    required: true
+    type: str
+  hostname:
+    description:
+      - The actual host to connect to when connecting to the host defined.
+    type: str
+  port:
+    description:
+      - The actual port to connect to when connecting to the host defined.
+    type: str
+  remote_user:
+    description:
+      - Specifies the user to log in as.
+    type: str
+  identity_file:
+    description:
+      - The path to an identity file (SSH private) that will be used
+        when connecting to this host.
+      - File need to exist and be 0600 to be valid.
+    type: str
+  user_known_hosts_file:
+    description:
+      - Sets the user known hosts file option.
+    type: str
+  strict_host_key_checking:
+    description:
+      - Whether to strictly check the host key when doing connections to the remote host.
+    choices: [ 'yes', 'no', 'ask' ]
+    type: str
+  proxycommand:
+    description:
+      - Sets the ProxyCommand option.
+    type: str
+  ssh_config_file:
+    description:
+      - SSH config file.
+      - If I(user) is none, C(/etc/ssh/ssh_config) is used.
+      - If I(user) is given, C(~/.ssh/config) is used.
+    type: str
+version_added: "2.0.0"
+requirements:
+- StormSSH
+notes:
+- Support check mode.
+'''
+
+EXAMPLES = r'''
+#
+'''
+
+RETURN = r'''
+#
+'''
+
+import os
+import traceback
+
+from copy import deepcopy
+
+STORM_IMP_ERR = None
+try:
+    from storm.parsers.ssh_config_parser import ConfigParser
+    HAS_STORM = True
+except ImportError:
+    HAS_STORM = False
+    STORM_IMP_ERR = traceback.format_exc()
+
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+
+
+class SSHConfig():
+    def __init__(self, module):
+        self.module = module
+        self.params = module.params
+        self.user = self.params.get('user')
+        self.group = self.params.get('group') or self.user
+        self.host = self.params.get('host')
+        self.config_file = self.params.get('ssh_config_file')
+        self.identity_file = self.params['identity_file']
+        self.check_ssh_config_path()
+        self.config = ConfigParser(self.config_file)
+        self.config.load()
+
+    def check_ssh_config_path(self):
+        if self.user:
+            if self.user == 'root':
+                self.config_file = '/etc/ssh/ssh_config'
```

Comment:
Hmm, this is strange. Why not use `/root/.ssh/config`, like for every other user as well?